src/test/java/org/eclipse/yasson/serializers/model/Trainer.java

```diff
@@ -0,0 +1,52 @@
+/*******************************************************************************
+ * Copyright (c) 2016, 2019 Oracle and/or its affiliates. All rights reserved.
```

Comment:
as soon as this is new file the only one copyright year shall be indicated here. I would choose year similar to creation year of the file.

Reply:
Fixed.